hw/bsp/same54xplainedpro/same54xplainedpro.c

```diff
@@ -166,15 +166,16 @@ void board_init(void)
 #endif
 
 	// LED0 init
```

Comment:
Seem like a typo, should be led pin than button

Reply:
Doh!